yarn.lock

```diff
@@ -3888,6 +3888,18 @@ beachball@^1.11.0:
     semver "^6.1.1"
     yargs-parser "^13.1.0"
 
+beachball@^1.11.5:
```

Comment:
Why is this additional beachball getting added to the yarn.lock instead of replacing the existing one?

Reply:
updated both refs now.